libraries/chain/transaction_context.cpp

```diff
@@ -828,7 +819,7 @@ namespace eosio { namespace chain {
                actors.insert( auth.actor );
          }
       }
-      EOS_ASSERT( one_auth || is_transient(), tx_no_auths, "transaction must have at least one authorization" );
+      EOS_ASSERT( one_auth || is_read_only(), tx_no_auths, "transaction must have at least one authorization" );
```

Comment:
Do we want to call out this in 5.0 Release Note as this is a backward incompatible change?

Reply:
dry-run trxs would fail when trying to record account cpu/net for first-authorizers. So unless protocol feature `only_bill_first_authorizer` was not enabled (it is enabled on all Antelope chains I know about) then it would fail with an unknown account error. I didn't test without `only_bill_first_authorizer`, so it is possible it would fail somewhere else even if the protocol feature was not enabled. This change just provides a much better error message.